proto/lavanet/lava/dualstaking/query.proto

```diff
@@ -16,10 +16,15 @@ service Query {
     option (google.api.http).get = "/lavanet/lava/dualstaking/params";
   }
 
-  // Queries a list of ListProjects items.
+  // Queries a delegator for all its providers.
 	rpc DelegatorProviders(QueryDelegatorProvidersRequest) returns (QueryDelegatorProvidersResponse) {
 		option (google.api.http).get = "/lavanet/lava/subscription/delegator_providers/{delegator}";
 	}
+
+  // Queries a provider for all its delegators.
+	rpc ProviderDelegators(QueryProviderDelegatorsRequest) returns (QueryProviderDelegatorsResponse) {
```

Comment:
indentation should be 2 spaces here.

Reply:
done. see https://github.com/lavanet/lava/commit/991a0e74722061319cbef1cd3199f4f893b87702